modules/yieldlabBidAdapter.js

```diff
@@ -186,13 +186,21 @@ export const spec = {
           // there may be publishers still rely on it
           bidResponse.adUrl = `${ENDPOINT}/d/${matchedBid.id}/${bidRequest.params.supplyId}/?ts=${timestamp}${extId}${gdprApplies}${gdprConsent}${pvId}`;
           bidResponse.mediaType = NATIVE;
-          const nativeImageAssetObj = find(matchedBid.native.assets, asset => isMainImage(asset));
+          const nativeIconAssetObj = find(matchedBid.native.assets, asset => isImageAssetOfType(asset, 1));
```

Comment:
Just a tiny thought: Having constants for the image asset types would allow us gett rid of the magic numbers and at the same time making the calling place even more specific.

`const nativeIconAssetObj = find(matchedBid.native.assets, asset => isImageAssetOfType(asset, ICON));`

Or, I know you intentionally made the previous `isMainImage` generic, we go back to having separate and specific methods for the different image asset types. Why? Because we could then have even more concise calls like e.g. `find(matchedBid.native.assets, iconAsset)`.

What do you think? At the end that is just cosmetics and not really mission critical but there is the potential that it just looks nicer :)